### PR TITLE
1188530751 - AADSU, on the locked table story, I can see the styling is different for cells in locked columns

### PR DESCRIPTION
**Pull request by frank-shenton**
dBefore submitting my PR to Code Review, I have made sure:

- [x] I have run the tests using `yarn run test`
- [x] I have exported any new files in index.js
- [x] My file supports Formik if will potentially be used in a form
- [x] The PR name follows the convention `(#<ticket_number>) - <ticket_user_story>`
- [x] I have incremented the version in `package.json` if I want to publish a new version
- [x] If my change is breaking I have updated the minor version number(i.e. the middle one) and don't merge until I have the PR on Corazon ready

### CODE REVIEW

File: src/components/organism/Table/Table.js [hunk 1]
```diff
@@ -70,14 +82,20 @@ export const Container = styled.div`
         flex-grow: 1;
       }
     }
-    :focus {
-      button {
-        opacity: 1;
-        cursor: pointer;
-        outline: none;
```
Comment (elio-stalteri) [inline src/components/organism/Table/Table.js:77]: why is this inside custom-disabled now?

File: src/components/organism/Table/Table.js [hunk 2]
```diff
@@ -168,6 +195,15 @@ export class TableWithTheme extends React.PureComponent {
...
         },
+        cellClass: (params) => {
+          const customCellClass = this.getCustomCellClass(params, column)
+
+          if (customCellClass && column.cellClass) {
+            return `${customCellClass} ${column.cellClass}`
+          }
+
+          return column.cellClass ?? customCellClass
+        },
```
Comment (elio-stalteri) [inline src/components/organism/Table/Table.js:206]: we are now using the other function to add class to the cell in ag-grid, can you refactor this? and make sure that the CDB still works as expected

File: src/components/organism/Table/Table.js (hunk 1 above)
Comment (frank-shenton) [reply src/components/organism/Table/Table.js:77]: It's not, I combined the `:hover button` and `:focus button` styling as it is the same. See lines 66-73

**Review by elio-stalteri — APPROVED** [review]

File: src/components/organism/Table/Table.js (hunk 2 above)
Comment (frank-shenton) [reply src/components/organism/Table/Table.js]: Good point. Have refactored to use `cellClassRules` like we are using on the CDB.

**Pull request merged**